frp/frp/service/signup.py

```diff
@@ -248,3 +248,37 @@ def create_donor_from_facebook(data):
 
     db.session.commit()
     return user
+
+def reopen(campaign):
+    new_target = campaign.target() - campaign.total_donations()
+    nbooks = int(round(new_target/40))
+    ###Find nbooks and nlic from new_target###########
```

Comment:
I have put this comment before our discussion on nlic & nbooks and i forgot to remove the comment.